pkg/logger/logger.go

```diff
@@ -120,6 +120,13 @@ func NewJSONLogger() (*ZapLogger, error) {
 	}, nil
 }
 
+func Must(logger *ZapLogger, err error) *ZapLogger {
```

Comment:
Instead of introducing this to be composed as `logger.Must(logger.NewTextLogger())`, I'd recommend just moving this up one level.

`MustNewTextLogger() *ZapLogger`
`MustNewJSONLogger() *ZapLogger`

then just use those directly 👍

Reply:
Done. Thanks!